.orchestra/ci/requirements.txt

```diff
@@ -44,6 +44,7 @@ pefile
 psutil
 pydot
 pyelftools
+python-idb
```

Comment:
Shouldn't we point this to our fork?

Reply:
Yes.

Reply:
OK, then you probably need to put the [URL of the git repo](https://www.activestate.com/resources/quick-reads/pip-install-git/). I guess your comment implied you knew that.

Reply:
But I already did it with: https://github.com/revng/orchestra/pull/95/commits/1f20a4893e1ab8d8ff1d23ed2de64a74f3ffe607. Is this enough?

Reply:
Ah OK, didn't notice. Mark as solved if done. Thanks! `:)`